amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpConnectorSettings.scala

```diff
@@ -211,8 +246,9 @@ final class AmqpWriteSettings private (
   private def copy(connectionProvider: AmqpConnectionProvider = connectionProvider,
                    exchange: Option[String] = exchange,
                    routingKey: Option[String] = routingKey,
-                   declarations: immutable.Seq[Declaration] = declarations) =
-    new AmqpWriteSettings(connectionProvider, exchange, routingKey, declarations)
+                   declarations: immutable.Seq[Declaration] = declarations,
+                   publishConfirm: Option[AmqpPublishConfirmConfiguration] = publishConfirm) =
+    new AmqpWriteSettings(connectionProvider, exchange, routingKey, declarations, publishConfirm)
 
   override def toString: String =
     "AmqpSinkSettings(" +
```

Comment:
Add the new property in `toString`. (And change it to `AmqpWriteSettings` while at it.)